roles/edpm_ovn/molecule/enable_chassis_gw_and_extport/verify.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+    - name: Include default tasks
+      ansible.builtin.include_tasks:
+        file: ../default/verify-tasks.yml
+
+    - name: Verify enable-chassis-as-gw and enable-chassis-as-extport-host
+      ansible.builtin.shell: >
+        /usr/bin/ovs-vsctl get open_vswitch . external_ids:ovn-cms-options
+      register: output
+      failed_when: output.stdout != "\"enable-chassis-as-gw,enable-chassis-as-extport-host\""
```

Comment:
(no action required) the order is probably of no importance, which makes the test a tiny bit more brittle; but I don't think it's a big deal and we can make it more robust if later we have to reshuffle the code / order.